### ISSUE TITLE
Training GAN with the network to improve detection and another question about the intuition of mixing RGB with IR

### ISSUE THREAD

**Issue by MoatazElTahan** [tdchaitanya/MMTOD#5]
Hi All,

I am using the intuition introduced in your paper in big research but after trying your technique I have some questions and concerns.

You mention in the following screen shot from the paper that to improve detection, you enable the training of GAN with the rest of the network but I have a problem with that,
when I enable the training of GAN the network loss is worse than the case of not training the GAN and load it with your provided pre-trained model.
So are there certain layers of the GAN, you are fine tuning or what maybe wrong in what I did?

![image](https://user-images.githubusercontent.com/50196337/125492977-68181679-6bad-494c-9e29-90a0c7918621.png)

Another concern if I freeze the GAN and load the pre trained model you are providing, the detection AP metrics at the end of training is the same comparing to using only the thermal branch and removing the RGB branch completely so it seems to me that the RGB branch doesn’t add any improvement to the detection task so is there a something I am missing that lead to these results.

I really appreciate any reply or direction from you about anything I may have done wrong.

Thanks in advance,
Moataz
